public/scripts/main.js

```diff
@@ -162,3 +150,20 @@ const gameHandler = (function gameIIFE() {
 })()
 
 stateManager.register('game', gameHandler)
+
+socket.on('setRole', setRole)
+
+function displayRole(role) {
```

Comment:
You can destructive this parameter into the three fields you need below. That's optional tho